platform-includes/getting-started-config/javascript.react.mdx

```diff
@@ -35,6 +38,10 @@ Sentry.init({
   replaysOnErrorSampleRate: 1.0,
 });
 
+// Make sure Sentry.init() is called before importing any of your other code
+import App from "./App";
+import { createRoot } from "react-dom/client";
```

Comment:
Aren't all imports are evalulated first, regardless where they appear in the file? Thought you would need to make a separate file for this, and import it first.

Reply:
It depends a bit on what the bundlers will do, but I think you are right. Vite seems to hoist the import statements up. I will update. Thanks!